contracts/NFTs/VehicleId.sol

```diff
@@ -170,6 +170,8 @@ contract VehicleId is Initializable, MultiPrivilege {
             }
         }
 
+        _dimoRegistry.onTransfer(from, to, tokenId);
```

Comment:
Could we call it something more specific with this name? Mention vehicle, at least?

Reply:
I will change this name. Was something temporary when I was talking to Juuso